### PR TITLE
Adds Red Dot support, letting folks know about availability in Folio

### PR DESCRIPTION
**Pull request by orta**
No joke. I dug back to a 2012 email to `team@art.sy` which spec'd out what we thought adding dots to artworks would look like if we had structured data for Editions. 

This PR is about working towards those designs, right now the grid will give you information about what artworks are sold or not. GIFs are a little out of date, there's been some polish since then, but the idea is sound

When you don't have editions (~95% of artworks at Artsy):

avail 2019-02-02 07_41_52

When you have edition info:

edition_avail 2019-02-02 07_39_38

When WIFI doesn't work:

offline

### CODE REVIEW

File: Classes/Views/Grid View & Items/ARGridViewCell.m [hunk 1]
```diff
@@ -99,24 +96,6 @@ - (ARDateLabel *)createSubtitleLabel
...
-    self.backgroundColor = backgroundColor;
-    self.contentView.backgroundColor = backgroundColor;
-    self.backgroundView.backgroundColor = backgroundColor;
-
-    self.titleLabel.backgroundColor = backgroundColor;
-    self.subtitleLabel.backgroundColor = backgroundColor;
-
-    self.titleLabel.textColor = foregroundColor;
-    self.subtitleLabel.textColor = foregroundColor;
-}
```
Comment (orta) [inline Classes/Views/Grid View & Items/ARGridViewCell.m]: This deletion is doing more damage than I thought, and needs to be brought back and handled correctly

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by orta** [comment]
The peril isn't a blocker, it's this: https://github.com/danger/danger-js/issues/807

File: Classes/Views/Grid View & Items/ARGridViewCell.m (hunk 1 above)
Comment (orta) [reply Classes/Views/Grid View & Items/ARGridViewCell.m]: I fixed this with the above green changes

**Comment by orta** [comment]
CI is fine, Peril is now. I'm ok with this.

**Pull request merged**